monitoring/configs/dashboards/logs.json

```diff
@@ -231,23 +250,18 @@
       },
       {
         "allValue": ".+",
-        "current": {
-          "selected": true,
-          "text": [
-            "All"
-          ],
-          "value": [
-            "$__all"
-          ]
+        "current": {},
```

Comment:
we are using fluxcd in different namespace in our project https://github.com/aenix-io/cozystack, so it would be better to rely on additional labels of release of FluxCD.